Add HCA publication email template in the SOPs (#1065)
ebi-ait/hca-ebi-wrangler-central: docs/SOPs/contributor_communication_SOP.md

[Template email](#first-contact---data-questionnaire){: .btn .btn-blue }

### Contact after an HCA publication email
Wrangler team might be CC'ed in an email informing the authors that a publication has been accepted as an HCA publication (subject __HCA Branded Publication__). In this case, we can contact them and do not wait for them to reach. 

> [Template email](#hca-branded-publication---first-contact){: .btn .btn-blue }

Reviewer: ```suggestion
[Template email- no preprint or publication available](#hca-branded-publication---no-preprint-or-publication){: .btn .btn-blue }
[Template email- preprint or publication available](#hca-branded-publication---with-preprint-or-publication){: .btn .btn-blue }
```

> 
> On behalf of the HCA DCP Data Wrangling Team

### HCA branded publication - no preprint or publication 

> Dear [name/s of contributor/s],
> 
> Congratulations on your HCA manuscript! We are looking forward to hearing more about your project, and will add your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication once a preprint or publication is available. 
> 
> Do you have any time-sensitive requirements on archiving, or have you already archived your data in a specific database?
> 
> Warm regards,
> [wrangler name] 
> 
> On Behalf of the HCA DCP Ingestion Service

> #### HCA branded publication - first contact

Reviewer: ```suggestion
#### HCA branded publication - with preprint or publication
```

> 
> On behalf of the HCA DCP Data Wrangling Team

### HCA branded publication - no preprint or publication 

> Dear [name/s of contributor/s],
> 
> Congratulations on your HCA manuscript! We are looking forward to hearing more about your project, and will add your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication once a preprint or publication is available. 
> 
> If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process.
> 
> Warm regards,
> [wrangler name] 
> 
> On Behalf of the HCA DCP Ingestion Service

#### HCA branded publication - with preprint or publication

> Dear [name/s of contributor/s],
> 
> Congratulations on your HCA manuscript!
> We have added the information about your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication.
> Due to internal processes it will take a day to display the publication information, in the meantime any feedback about the HCA project catalogue is greatly appreciated!
> 

> > If you have already archived your data in any database, could you provide us the accession number? If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process. Also, please inform us for any time-sensitive requirements on archiving.

Reviewer: I'd separate the 2 ideas here in 2 different paragraphs:
- First one, about asking for a pointer to the data
- Second one, offering to wrangle their data to the HCA data portal

### HCA branded publication - no preprint or publication 

> Dear [name/s of contributor/s],
> 
> Congratulations on your HCA manuscript! We are looking forward to hearing more about your project, and will add your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication once a preprint or publication is available. 
> 
> If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process.
> 
> Warm regards,
> [wrangler name] 
> 
> On Behalf of the HCA DCP Ingestion Service

#### HCA branded publication - with preprint or publication

> Dear [name/s of contributor/s],
> 
> Congratulations on your HCA manuscript!
> We have added the information about your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication.
> Due to internal processes it will take a day to display the publication information, in the meantime any feedback about the HCA project catalogue is greatly appreciated!
> 
> If you have already archived your data in any database, could you provide us with the accession number?
> 

> > If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process. Also in that case, please inform us of any time-sensitive requirements on archiving.

Reviewer 2: ```suggestion
> If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process. Also in that case, please inform us of any time-sensitive requirements.
```

Reviewer 2: If they already deposited their data in an archive there is no need for us to offer archiving. 
We are unable to archive right now anyway so there is no point in mentioning it
